kyo-prelude/shared/src/main/scala/kyo/kernel/Pending.scala

```diff
@@ -45,6 +45,14 @@ object `<`:
         ): B < (S & S2) =
             map(_ => f)
 
+        inline def as[B, S2](other: B < S2)(
```

Comment:
I noticed that the kernel still doesn't have scaladocs. I'll follow up separately.